Sources/NIOFileSystem/Internal/System Calls/Syscall.swift

```diff
@@ -313,6 +313,24 @@ public enum Libc {
     }
     #endif
 
+    #if canImport(Darwin)
+    @_spi(Testing)
+    public static func copyfile(
```

Comment:
Can you add a test for this please? https://github.com/apple/swift-nio/blob/ba6245748fe1c07d72bd486fb056deced5baad21/Tests/NIOFileSystemTests/Internal/SyscallTests.swift#L376?